integration/single-node-slug-deploy/check.go

```diff
@@ -348,3 +365,29 @@ func verifyHelloRunning() error {
 		return nil
 	}
 }
+
+func verifyHealthChecks(config *preparer.PreparerConfig, services []string) error {
+	// do consulGet: key = hello
+	opts := kp.Options{
+		Address: config.ConsulAddress,
+		HTTPS:   false,
+	}
+	store := kp.NewConsulStore(opts)
+
+	time.Sleep(5)
+	// check consul for health information for each app
+	for _, sv := range services {
+		res, err := store.GetHealth(sv, "localhost.localdomain")
+		if err != nil {
+			return err
+		} else if res == "" {
+			fmt.Errorf("No results for %s", sv)
```

Comment:
This doesn't print anything - it just returns an `error` for use. You need to actually `Print` it.